AllReadyApp/Web-App/AllReady/Models/Activity.cs

```diff
@@ -7,6 +7,13 @@ namespace AllReady.Models
 {
     public class Activity
     {
+        public Activity()
+        {
+            Tasks = new List<AllReadyTask>();
```

Comment:
Since we're using all the latest, I'd like to see us use the C# 6 initialization syntax:
e.g.
Tasks { get; set; } = new List<AllReadyTask>();